src/transformState.js

```diff
@@ -62,7 +62,24 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const transform of transforms) {
+    switch (transform.operation) {
+      case 'addProperties':
+        Object.assign(state, transform.properties);
+        break;
+
+      case 'removeProperties':
+        for (const keys of transform.properties) {
+          delete state[keys];
+        }
+        break;
+
+      case 'clear':
+        for (const value in state) {
```

Comment:
```suggestion
        for (const key in state) {
```
так как в объектах перебор идет по ключам, лучше называть key, так как value - это обычно то, что хранится в ключе

Reply:
fix this suggestion